azure-functions-maven-plugin/src/main/java/com/microsoft/azure/maven/function/AbstractFunctionMojo.java

```diff
@@ -11,29 +11,33 @@
 import com.microsoft.azure.maven.AbstractAppServiceMojo;
 import com.microsoft.azure.maven.appservice.PricingTierEnum;
 import com.microsoft.azure.maven.auth.AzureAuthFailureException;
+import org.apache.maven.plugin.MojoExecutionException;
 import org.apache.maven.plugins.annotations.Parameter;
 
 import javax.annotation.Nullable;
 import java.io.File;
 
 public abstract class AbstractFunctionMojo extends AbstractAppServiceMojo {
-    //region Properties
 
+    private static final String JDK_VERSION_ERROR = "Your JDK Version is %s ,which doesn't fit azure maven " +
```

Comment:
Local JDK version %s does not meet the maven plugin requirement. The supported version is JDK 8